src/promptflow-evals/tests/evals/e2etests/README.md

```diff
@@ -0,0 +1,37 @@
+# E2E tests logging to the application insights.
+For the regression tracking we are recording test run times and estimate the installation times of promptflow-evals wheel with and without `azure` optional dependency. The table below summarizes the events we are logging.
+
+## Events
+| Event | Description | Logging value | Is recorded |
+|-------|-------------|---------------|-------------|
+| install_promptflow_no_extras | Installation time of the packages without Azure | The number of packages installed and installation time in seconds | N/A |
+| install_promptflow_with_extras | Installation time of the packages with Azure | The number of packages installed and installation time in seconds | N/A |

```

Comment:
I think this should be yes. For local tests, it doesn't use VCR recording but it uses recordings of aoai traffic.

Reply:
Fixed.